projects/packages/status/src/class-modules.php

```diff
@@ -198,6 +198,10 @@ public function get_active() {
 			$active[] = 'protect';
 		}
 
+		// If it's not available, it shouldn't be active.
+		// We don't delete it from the options though, as it will be active again when a plugin gets reactivated.
+		$active = array_intersect( $active, $this->get_available() );
```

Comment:
This is causing the [test_sync_deactivate_module_event](https://github.com/Automattic/jetpack/blob/017c22e95d1a611cd0cd485872a2b9ef855d5e37/projects/plugins/jetpack/tests/php/sync/test_class.jetpack-sync-modules.php#L22) test to fail, since it's trying to deactivate a module that isn't available (`stuff`). Should we wrap this in a `! defined( 'PHPUNIT_JETPACK_TESTSUITE' )` or something?

Reply:
Tried simply using a valid module slug in the test... I think it works, let's wait for the run